PhysicsTools/NanoAOD/python/nano_cff.py

```diff
@@ -371,6 +395,7 @@ def nanoAOD_customizeCommon(process):
                                      addParticleNet=nanoAOD_addDeepInfoAK8_switch.nanoAOD_addParticleNet_switch,
                                      jecPayload=nanoAOD_addDeepInfoAK8_switch.jecPayload)
     (run2_nanoAOD_94XMiniAODv1 | run2_nanoAOD_94X2016 | run2_nanoAOD_94XMiniAODv2 | run2_nanoAOD_102Xv1 | run2_nanoAOD_106Xv1).toModify(process, lambda p : nanoAOD_addTauIds(p))
```

Comment:
This is not introduced by you, but why are these modifiers needed here?

Reply:
As far as I know, this is to produce IDs on top of miniAOD input, which were not yet part of miniAOD at that time but requested for nanoAOD. Meanwhile, those are obtained from miniAOD directly.